packaging/docs-weapons.py

```diff
@@ -0,0 +1,50 @@
+import argparse
+import json
+from collections import OrderedDict
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--input', type=str, required=True)
```

Comment:
Remove the `--` here and in the line below, then call this as `python3 ./packaging/docs-weapons.py docs/api/release/weapons.json docs/api/release/weapons.md`